src/room.ts

```diff
@@ -204,6 +210,16 @@ export class Room extends EventEmitter implements Sayable {
       return null
     }
 
+    if (!rawObj.MemberList) {
```

Comment:
Get rid of this `if`, and also get rid of `return` inside it.

We should prevent to use multiple return statement for the final data.

Only return the final data at the end of function statements. The "return"s  before that should only return for the error conditions.